src/jarabe/model/filetransfer.py

```diff
@@ -54,7 +54,6 @@
 FT_REASON_LOCAL_STOPPED = 2
 FT_REASON_REMOTE_STOPPED = 3
 FT_REASON_LOCAL_ERROR = 4
-FT_REASON_LOCAL_ERROR = 5
```

Comment:
5 is valid.  See https://lazka.github.io/pgi-docs/TelepathyGLib-0.12/enums.html#TelepathyGLib.FileTransferStateChangeReason

Also occurs in collabwrapper/collabwrapper.py

Reply:
In that documentation REMOTE_ERROR= 5 and LOCAL_ERROR= 4

Reply:
Please review https://github.com/sugarlabs/collabwrapper/pull/24